icicle/backend/cpu/include/ntt_cpu.h

```diff
@@ -213,6 +326,7 @@ namespace ntt_cpu {
       E* current_elements =
         ntt_data.config.columns_batch ? ntt_data.elements + batch : ntt_data.elements + batch * ntt_data.size;
 
+      // #pragma omp parallel for
```

Comment:
are you keeping those?

Reply:
Removed the OpenMP comments and implemented parallelism using TaskFlow instead